doc/api/Tools/MediaCapabilitiesProber.md

```diff
@@ -144,40 +122,26 @@ const mksConfiguration = {
   ],
 };
 
-const keySystems = [
-  // Let's consider this one as a compatible key system configuration
-  { type: "com.widevine.alpha", configuration: mksConfiguration },
-
-  // Let's consider this one as not compatible
-  { type: "com.microsoft.playready", configuration: mksConfiguration },
-];
-
-mediaCapabilitiesProber.getCompatibleDRMConfigurations(keySystems).then((drmConfigs) => {
-  drmConfigs.forEach((config) => {
-    const { type, configuration, compatibleConfiguration } = config;
-
-    if (compatibleConfiguration !== undefined) {
-      console.log("# Compatible configuration #############################");
-      console.log("Key System:", type);
-      console.log("Wanted configuration:", configuration);
-      console.log("Compatible configuration:", compatibleConfiguration);
-      console.log("########################################################");
-      console.log("");
-    } else {
-      console.log("# Incompatible configuration ###########################");
-      console.log("Key System:", type);
-      console.log("Wanted configuration:", configuration);
-      console.log("########################################################");
-      console.log("");
-    }
-  });
+mediaCapabilitiesProber.checkDrmConfiguration(
+  "com.widevine.alpha",
+  mksConfiguration,
+  ).then(
+  // On success:
+  (config) => {
+    console.log("This device is compatible with the given configuration!");
+    console.log("Wanted configuration:", configuration);
+    console.log("Compatible configuration:", compatibleConfiguration);
```

Comment:
this example is not working, `configuration` and `compatibleConfiguration` are not defined

Reply:
You're right, it's directly the compatible one instead. I'll update.